Program/program-12.py

```diff
@@ -0,0 +1,27 @@
+f=open("C:\\23bca273\\sqlite3\\data\\bca.txt","w")
+f.write("bca is it cose \n")
+f.write("bca is corently high dimand \n")
+f.write("bca is used for internet \n")
+f.write("bca is cyber cisyority \n")
+f.write("bca is to developer \n")
+f.close()
+f=open("C:\\23bca273\\sqlite3\\data\\bca.txt","r")
+l=f.read()

```

Comment:
Why you use " l " variable to read the file ??

Reply:
this  is used to store content